doc/source/development/contributing_environment.rst

```diff
@@ -162,11 +118,8 @@ We'll now kick off a three-step process:
 .. code-block:: none
 
    # Create and activate the build environment
-   conda env create -f environment.yml
-   conda activate pandas-dev
-
-   # or with older versions of Anaconda:
-   source activate pandas-dev
+   mamba env create -f environment.yml
```

Comment:
The `-f environment.yml` is not needed. I remember someone wanted to have it anyway at that time (not sure why), but if that's not the case anymore, I'd remove it and make things simpler.